src/components/Sidebar/SidebarNavItem/SidebarNavItem.tsx

```
@@ -0,0 +1,138 @@
+      className={({ isActive }) =>
+        useSimpleButton
+          ? isActive
+            ? styles.leftDrawerActiveButton
+            : styles.leftDrawerInactiveButton
+          : isActive
+            ? styles.sidebarBtnActive
+            : styles.sidebarBtn
+      }
+      data-testid={testId}
+      style={useSimpleButton ? { height: '40px' } : undefined}
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)